app/App.js

```diff
@@ -50,6 +50,7 @@ class App extends Component<{
   render() {
     const { stores, actions, history } = this.props;
     const locale = stores.profile.currentLocale;
+    const version = stores.profile.lastLaunchVersion;
```

Comment:
Not used?

Reply:
I had to include at least one call to lastLaunchVersion so the computed value gets its value, otherwise is undefined. Other than that, it is not used.

Reply:
Update: there is no need to use it after the changes I made 👍